cst438_register/src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -0,0 +1,99 @@
+package com.cst438.controller;
```

Comment:
Is this class completely new or something? It says it was previously an empty file.

Reply:
Yes, the StudentController class was created by me to separate it from other controllers.